main.js

```diff
@@ -0,0 +1,15 @@
+const p = document.querySelector("#sec");
+const storage = window.localStorage;
+
+let i = storage.getItem("timeinsec");
+if(!i) {
+	i = 0;
+	storage.setItem("timeinsec", 0);
+}
+
+p.innerHTML = i;
+setInterval(() => {
+	i++;
+	p.innerHTML = i;
```

Comment:
you duplicated this row. Better will create a function for increment `i` and for render `i`.